### PR TITLE
Features/clarify message naming 177909992

### PR DESCRIPTION
**Pull request by rosalinekarr**
Problem
=======
We use the terms "announcement," "message," "content," and "event" interchangeably for several different concepts which may be confusing for people reading the spec.
[#177909992](https://www.pivotaltracker.com/story/show/177909992)

Solution
========
Change our usage of these terms to be consistent and add a section to the message overview document to expand on their meanings.

### CODE REVIEW

File: pages/Messages/Overview.md [hunk 1]
```diff
@@ -13,26 +13,57 @@ menu: Messages
...
+
+An announcement refers specifically to a DSNP item as described in this specification document.
+An announcement is intended for inclusion in a batch file and to eventually be posted to the blockchain via a batch log event.
+Some announcements, such as broadcasts and replies, will contain references to content items.
+
+#### Content
+
+Content items, or simply content, refers to an activity pub compliant object hosted at some URI and intended to be posted via an announcement in a batch.
+Generally, content within the DSNP specification will be defined by the [W3C Activity Pub specification](https://www.w3.org/TR/activitypub/#object-without-create), however extensions, such as the [Mastadon Activity Pub specification](https://docs.joinmastodon.org/spec/activitypub/) may also be implemented.
+This specification will not directly define any standards for content.
```
Comment (wilwade) [inline pages/Messages/Overview.md:47]: Hmmm... This feels a bit wrong as just above this we specify it as matching the W3C Activity Pub. I think we do need to "define" the standards, just not in this page (which might be actually what "this spec" means.)

**Review by wilwade — CHANGES_REQUESTED** [review]
One line I am confused by, but this is much clearer. 💯

File: pages/Messages/Overview.md [hunk 2]
```diff
@@ -13,26 +13,57 @@ menu: Messages
 | 0.2     | Tentative |
```
Comment (wilwade) [inline pages/Messages/Overview.md:13]: @rosalinekarr P.S. We should increment the version number.

**Review by wilwade — COMMENTED** [review]
One more comment.

File: pages/Messages/Overview.md (hunk 1 above)
Comment (rosalinekarr) [reply pages/Messages/Overview.md]: Removed line.

File: pages/Messages/Overview.md (hunk 2 above)
Comment (rosalinekarr) [reply pages/Messages/Overview.md]: Fixed

**Review by wilwade — APPROVED** [review]
Might be good for @shannonwells to review as well.

**Comment by enddynayn** [comment]
This PR is amazing! This is very helpful.

**Review by enddynayn — APPROVED** [review]

File: .spelling [hunk 3]
```diff
@@ -97,3 +97,6 @@ roadmap
 NodeJS
 Lookups
 ICANN
+contentHash
+W3C
+Mastadon
```
Comment (shannonwells) [inline .spelling:102]: it's Mastodon https://mastodon.social/about

**Review by shannonwells — APPROVED** [review]
Thanks for doing this! Approve assuming the spelling correction.

**Pull request merged**